DataFormats/L1Trigger/src/classes_def.xml

```diff
@@ -278,4 +278,10 @@
   <class name="edm::Wrapper<l1t::L1DataEmulResultBxCollection>"/>
 
   <class name="edm::Ptr<l1t::L1Candidate>"/>
+
+  <class name="l1t::Vertex"/>
+  <class name="std::vector<l1t::Vertex>"/>
+  <class name="l1t::VertexCollection"/>
```

Comment:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b282c5/15024/dupDict/lostDefs.txt

```
Searching for 'classes_def.xml' in '/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_0_X_2021-05-11-2300/src'.

./DataFormats/L1Trigger/src/classes_def.xml
  Vertex : l1t::Vertex
  Vertex : vector< l1t::Vertex >
```

Looking at the example in https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Create_a_classes_def_xml_in_the
I think you have to remove `  <class name="l1t::VertexCollection"/>` as it is already defined in `  <class name="std::vector<l1t::Vertex>"/>`

Please note that in the other l1t data formats there is the `BxCollection` version of the data format. In this case it would be `l1t::VertexBxCollection`, but I see that it is not implemented in the code.